src/Cake.Core/Scripting/CodeGen/PropertyAliasGenerator.cs

```diff
@@ -150,7 +160,25 @@ private static string GenerateCode(MethodInfo method)
             return builder.ToString();
         }
 
-        private static string GenerateCachedCode(MethodInfo method)
+        private static string GenerateCommonInitalCode(MethodInfo method, StringBuilder builder)
```

Comment:
Can we make pass `StringBuilder` using `ref`? Not that it matters but indicates to the reader that we're mutating it inside the method.